modules/tensor_mechanics/doc/content/source/bcs/Pressure.md

```diff
@@ -19,6 +19,127 @@ name.
 A set of +`Pressure`+ boundary conditions applied to multiple variables in multiple
 components can be defined with the [PressureAction](/BCs/Pressure/index.md).
 
+### Jacobian
+
+#### 3D
+
+Let $N$ be the number of nodes on a finite element face.  Also, let $h_i$ be the shape function at node $i$.  Then the vector $f$, which has dimension $3N\times1$ for a 3D model, is defined as
+\begin{equation}
+  f = \int \gamma F^T n\;\; dA
+\end{equation}
+where $\gamma$ represents a scaling factor, $n$ is the normal vector, and
+\begin{equation}
+  F = \begin{bmatrix}
+      {h_1}I & {h_2}I & ... & {h_N}I
+      \end{bmatrix}
+\end{equation}
+where $I$ is the identity tensor.
+
+To find the Jacobian, we take the variation of the term in the integral,
+\begin{equation}
+  \delta(\gamma F^T n) = \gamma(\delta(F^T)n + F^T\delta(n))
+\end{equation}
+
```

Comment:
How about the `area` term `A`? If `A` refers to the deformed configuration, there will be a Jacobian term for that, correct?

Reply:
Should there be?  If so, I don't have it.  I'm having a hard time seeing it.  We don't have something like that for stress divergence, for example.  Am I missing something?

Reply:
I just pushed changes to the documentation file that I think clarify the situation.  The changes are mostly in the section on spherical symmetry and axisymmetric cases.